storage/src/tests/distributor/bucketdbupdatertest.cpp

```diff
@@ -1835,10 +1841,11 @@ BucketDBUpdaterTest::mergeBucketLists(
     }
 
     BucketDumper dumper(includeBucketInfo);
-    getBucketDBUpdater().getDistributorComponent()
-            .getBucketDatabase().forEach(dumper);
-    getBucketDBUpdater().getDistributorComponent()
-            .getBucketDatabase().clear();
+    auto &bucketSpaceRepo(getBucketDBUpdater().getDistributorComponent().getBucketSpaceRepo());
+    auto &distributorBucketSpace(bucketSpaceRepo.get(makeBucketSpace()));
```

Comment:
These common lines could be factored out into a `defaultDistributorBucketSpace()` or similarly named test member function